contracts/StaderOracle.sol

```diff
@@ -119,6 +119,9 @@ contract StaderOracle is IStaderOracle, AccessControlUpgradeable, PausableUpgrad
         if (block.number < lastTrustedNodeCountChangeBlock + trustedNodeChangeCoolingPeriod) {
             revert CooldownNotComplete();
         }
+        if (trustedNodesCount <= MIN_TRUSTED_NODES) {
```

Comment:
Make the threshold for SD price update logic similar to every other update please.

Reply:
[2d6a43f](https://github.com/stader-labs/ethx/pull/258/commits/2d6a43ff9e077c383e0583fef64b870d9d9c2520)